src/display/editor/freetext.js

```diff
@@ -386,13 +386,16 @@ class FreeTextEditor extends AnnotationEditor {
    * @returns {string}
    */
   #extractText() {
-    const divs = this.editorDiv.getElementsByTagName("div");
-    if (divs.length === 0) {
-      return this.editorDiv.innerText;
-    }
+    // We don't use innerText because there are some bugs with line breaks.
     const buffer = [];
-    for (const div of divs) {
-      buffer.push(div.innerText.replace(/\r\n?|\n/, ""));
+    this.editorDiv.normalize();
+    const nodes = this.editorDiv.childNodes;
+    const EOL_PATTERN = /\r\n?|\n/g;
+    for (let i = 0, ii = nodes.length; i < ii; i++) {
+      const child = nodes[i];
```

Comment:
Can we use for...of iteration instead here, and "simply" this as follows?
```suggestion
    const EOL_PATTERN = /\r\n?|\n/g;
    for (const child of this.editorDiv.childNodes) {
```